### DISCUSSION TITLE
More than one Moodle user?

### DISCUSSION THREAD

**Discussion by OrionCodeDev** [tjarbo/discord-moodle-bot#114] [Q&A]
Hello, are there any plans to create a feature that would allow more than one Moodle account to be added? A simple example at my university, students are divided into 6 different groups, despite the fact that it is the same major. There are just too many of us. We have mixed teachers, e.g. we have maths divided into two groups, each with a different teacher. A nice option would be if you could add more than one Moodle account, e.g. 1 for each group of students. Best Regards

**Comment by tjarbo** [comment]
Hey! yes, I see the use case for that. Thank you for the example! Even it is not listed on our feature roadmap, we already had such an implementation in mind (I will add this to the list later!). But right now, we are focused on implementing and improving the notification services to get more independent of Discord, as you maybe already have read at the README. 

Right now, it is unfortunately only possible to have multiple instances of fmdb running right now, each with its own user account.

**Comment by OrionCodeDev** [comment]
Hey, any update :D?

**Reply by tjarbo** [reply]
Hey @ChronoBrake, unfortunately not really. For the last release our focus was to reimplement the "connectors" to add more than just a discord bot. Therefore we want to add at least one more connector. Afterwards our next step is to improve the moodle implementation and file detection. These changes will already cover the basic implementation to suppport multiple moodle accounts, which "management" than just has to be implemented.

So it will need some time, but thanks to your contribution we added the new requirement to support different moodle accounts with the new implementation 👍

**Reply by FreshImmuc** [reply]
I would have a lot of use cases for that too. Would be great to have that!

**Reply by tjarbo** [reply]
I want to add the support of mutiple moodle instances/users as part of a rewrite of the moodle analysis. But this is a huge feature as the ui has to be adapted as well.
